Mailnag/configuration/accountdialog.py

```diff
@@ -39,6 +39,9 @@
 IDX_YAHOO	= 3
 IDX_IMAP	= 4
 IDX_POP3	= 5
+IDX_MBOX	= 6
+IDX_MAILDIR	= 7
+
 
 PROVIDER_CONFIGS = [
 	[ 'Gmail', 'imap.gmail.com', '993'],
```

Comment:
Refactoring idea: These provider configs could be replaced by adding them to the backends map with suitable defaults in `Mailnag/backends/__init__.py`. They all could use IMAPMailboxBackend class. Then account dialog does not need to handle them specially.